cinema/models.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField(max_length=255)
```

Comment:
if you want to specify `max_length` it's more reasonable to use CharField here

Reply:
Understood, thank you!